op-chain-ops/genesis/config_test.go

```diff
@@ -26,6 +26,20 @@ func TestConfigMarshalUnmarshal(t *testing.T) {
 	require.JSONEq(t, string(b), string(encoded))
 }
 
+func TestConfigWithExtraDataMarshalUnmarshal(t *testing.T) {
```

Comment:
What do you think about not adding an additional json file for this and instead updating an existing json file?

Reply:
Yeah this could be overkill.  I created a separate one so I could have both default and non-default code paths getting exercised but the code is pretty simple so probably not so important.

Reply:
Have consolidated it to a single test.

Reply:
Thank you!